### PR TITLE
modified K3s install section of Readme.md 

### PR DESCRIPTION
**Pull request by colin-mccarthy**
Changed up the k3s install section and added an ingress section.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -57,33 +57,38 @@ $ kubectl apply -f manifests/ # This command sometimes may need to be done twice
...
-4. Set _traefikExporter_ `enabled` parameter to `true` to collect Traefik metrics and deploy dashboard.
+If you get an error from applying the manifests, run the `make deploy` or `kubectl apply -f manifests/` again. Sometimes the resources required to apply the CRDs are not deployed yet.
 
-After changing these values, run `make` to build the manifests and `k3s kubectl apply -f manifests/` to apply the stack to your cluster. In case of errors on some resources, re-run the command.
+## Ingress 
 
 Now you can open the applications:
 
-To list the created ingresses, run `k3s kubectl get ingress --all-namespaces`.
+To list the created ingresses, run `kubectl get ingress --all-namespaces`, you will notice they need to be updated to include `your_node_ip`.
```
Comment (carlosedp) [inline Readme.md]: If you edit `vars.jsonnet` adding the correct IP for your infress before running `make` and deploying the manifests, the ingresses are created correctly.
No need to update them after deployment. Maybe clarify this.

**Comment by carlosedp** [comment]
Hey @colin-mccarthy, can you allow maintainers to add commits. I'll just adjust a few things and merge.
Thanks

**Comment by colin-mccarthy** [comment]
Hey @carlosedp I have allowed maintainers to add commits. Thanks!

**Comment by carlosedp** [comment]
I've merged your PR... added a commit adjusting a couple parts.

**Pull request merged**